src/systemtask/SystemTask.cpp

```diff
@@ -199,10 +212,14 @@ void SystemTask::Work() {
             spiNorFlash.Sleep();
           }
           lcd.Sleep();
-          touchPanel.Sleep();
-
           spi.Sleep();
-          twiMaster.Sleep();
+
+          // Double Tap needs the touch screen to be in normal mode
+          if ( settingsController.getWakeUpMode() != Pinetime::Controllers::Settings::WakeUpMode::DoubleTap ) {
+            touchPanel.Sleep();
```

Comment:
I thought that the touch panel would automatically wake up (and send an IRQ) when it was touched, even in sleep mode. Do you confirm sleep mode disabled double tap detection?

Reply:
In sleep mode he continues to send the irq but afterwards we are unable to know what the type of event was, if he is sleeping when doing the reset to wake we lost that information.